trunk/workflow-manager/src/test/java/org/mitre/mpf/wfm/camelOps/TestRemoteMediaProcessor.java

```diff
@@ -52,13 +53,17 @@
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import javax.annotation.PostConstruct;
-import java.util.*;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
 
 @ContextConfiguration(locations = {"classpath:applicationContext.xml"})
 @RunWith(SpringJUnit4ClassRunner.class)
 public class TestRemoteMediaProcessor {
 	private static final Logger log = LoggerFactory.getLogger(TestRemoteMediaProcessor.class);
 	private static final int MINUTES = 1000*60; // 1000 milliseconds/second & 60 seconds/minute.
+	private static final String EXT_IMG = "https://raw.githubusercontent.com/openmpf/openmpf/develop/trunk/mpf-system-tests/src/test/resources/samples/face/meds-aa-S001-01.jpg";
```

Comment:
I think it's better if we look at the master branch instead of the develop branch. That way we don't have to update the master branch if the layout of files in the develop branch changes.